test/controllers/update/update.beneficial.owner.type.controller.spec.ts

```diff
@@ -76,6 +78,33 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
 
   describe("GET tests", () => {
 
+    test(`render the ${config.UPDATE_BENEFICIAL_OWNER_TYPE_PAGE} page with table of reviewed BOs if BOs have been reviewed`, async () => {
+      const reviewBoAppData = { ...APPLICATION_DATA_UPDATE_BO_MOCK };
+      delete reviewBoAppData["beneficial_owners_individual"];
+      reviewBoAppData["beneficial_owners_individual"] = [UPDATE_REVIEW_BENEFICIAL_OWNER_INDIVIDUAL_OBJECT_MOCK];
+
+      mockGetApplicationData.mockReturnValueOnce({
+        ...reviewBoAppData
+      });
+
+      const resp = await request(app).get(config.UPDATE_BENEFICIAL_OWNER_TYPE_URL);
+
+      expect(resp.status).toEqual(200);
+      expect(resp.text).toContain(BENEFICIAL_OWNER_MANAGING_OFFFICER_TYPE_PAGE_HEADING);
+      expect(resp.text).toContain(REVIEWED_BENEFICIAL_OWNER_MANAGING_OFFICER_TABLE_HEADING);
+    });
+
+    test(`renders the ${config.UPDATE_BENEFICIAL_OWNER_TYPE_PAGE} page without review BO table if no BOs have been reviewed`, async () => {
+      mockGetApplicationData.mockReturnValueOnce({
+        [BeneficialOwnerStatementKey]: BeneficialOwnersStatementType.ALL_IDENTIFIED_ALL_DETAILS,
+        ...APPLICATION_DATA_UPDATE_BO_MOCK });
```

Comment:
This test might fail if I merge first as I think I've changed the ch_reference value in a few mocks. Just a heads up